tests/test_backpack.sh

```diff
@@ -21,23 +21,22 @@ function functional_test_backpack {
   figlet $(basename $0)
   apply_operator
   kubectl apply -f tests/test_crs/valid_backpack.yaml
-  uuid=$(get_uuid 20)
+  long_uuid=$(get_uuid 20)
+  uuid=${long_uuid:0:8}
 
   wait_for_backpack $uuid
+  
+  echo "Waiting 2 minutes to ensure data is in ES"
+  sleep 120
 
-  for pod in `kubectl -n my-ripsaw get pods -l name=backpack-$uuid | grep -v NAME | awk '{print $1}'`
-  do
-    if [[ `kubectl -n my-ripsaw exec $pod -- ls /tmp/stockpile.json` ]]
-    then
-      echo "Found stockpile.json on "$pod
-    else
-      echo "Could not find stockpile.json on "$pod
-      echo "Backpack test: Fail"
-      exit 1
-    fi
-  done
-
-  echo "Backpack test: Success"
+  indexes="cpu_vulnerabilities-metadata cpuinfo-metadata dmidecode-metadata k8s_configmaps-metadata k8s_namespaces-metadata k8s_nodes-metadata k8s_pods-metadata lspci-metadata meminfo-metadata sysctl-metadata"
```

Comment:
this would mean everytime we add a new scribe module, we'll have to update this... im okay with it, but shouldn't backpack be tested outside of ripsaw? we are still testing its integration with ripsaw ( we've to wait for benchmark pods to come up, which in turn waits on backpack to finish )

Reply:
We already have a test backpack script/cr in place. This was just adding to the robustness of the test. Additionally, this would ensure that errors related to the backpack role/permissions/etc get caught.